modules/xfem/doc/content/modules/xfem/theory/theory.md

```diff
@@ -1,4 +1,127 @@
-# XFEM Theory Manual
+# XFEM Module Theory
 
-!alert construction
-The XFEM module Theory Page needs documentation.
+The MOOSE XFEM module is a library for the implementation of extended finite element (XFEM) that solve partial differential equations (PDEs) with some form of discontinuity. For an overview of the XFEM fracture capability, please see the journal article [!cite](jiang2020).
+
+## Phantom-node-based XFEM
+
+The XFEM approach as originally proposed [!cite](belytschko_elastic_1999, Dolbow99) enriches the standard continuous finite element interpolation of the solution field $\boldsymbol{u}$ as a function of spatial coordinate $\boldsymbol{x}$ and time $t$ with Heaviside and near-tip enrichment functions:
+\begin{equation}
+\boldsymbol{u}(\boldsymbol{x},t) = \underbrace{\sum_{I=1}^n N_I(\boldsymbol{x}) \boldsymbol{u}_I(t)}_{\text{Continuous}}
+                 + \underbrace{\sum_{J=1}^n N_J(\boldsymbol{x}) H(\boldsymbol{x})\boldsymbol{b}_J(t)}_{\text{Heaviside Enriched}}
+                 + \underbrace{\sum_{K=1}^n N_K(\boldsymbol{x}) \sum_{L=1}^4 F_L(\boldsymbol{x})c_K^L(t)}_{\text{Near-tip Enriched}},
+\end{equation}
+where $n$ is the number of nodes per finite element, $N_I$ are the standard finite element interpolation functions, $\boldsymbol{u}_I$ are the nodal displacements, $H$ is the Heaviside function, $\boldsymbol{b}_J$ are additional nodal degrees of freedom corresponding to Heaviside enrichment, $F_L$ are a set of enrichment functions that capture asymptotic near-tip fields, and $\boldsymbol{c}_K$ are additional nodal degrees of freedom corresponding to the near-tip enrichment. This approach requires an additional degree of freedom per solution variable for each node whose support is traversed by a crack for Heaviside enrichment, as well as 4 additional degrees of freedom per solution variable for the nodes in the vicinity of the crack tip to which near-tip enrichment is applied. The set of enriched nodes is typically a small subset of the nodes in the discretized solution because a small fraction of the solution domain is typically directly affected by cracking. As a result, this does not drastically increase the number of unknowns in the system of equations. However, this can be difficult to implement in existing FEM codes if they do not permit nodes to have arbitrary number of degrees of freedom.
```

Comment:
Does it make sense to explicitly point out XFEM's locally enriched nature here? It's a big part of what separates it from methods such as generalized FEM (GFEM), right?